rlberry/manager/agent_manager.py

```diff
@@ -233,7 +235,7 @@ class AgentManager:
     def __init__(
         self,
         agent_class,
-        train_env,
+        train_env=(None, None),
```

Comment:
Minor: I see you have added typing somewhere else in the same file, maybe we can add them here as well? But it's ok, we can do a full pass afterward, not important.